masterlist.yaml

```
@@ -15646,12 +15648,16 @@ plugins:
       - 'Original Opening Scene.esp'
       - 'QuickVanillaStart.esl'
       - 'Random Alternate Start Reborn SE.esp'
-      - 'Realm of Lorkhan - Custom Alternate Start - Choose your own adventure.esp'
       - 'Skyrim Unbound.esp'
     msg:
       - <<: *patchUnavailable
         subs: [ 'Touring Carriages' ]
         condition: 'active("TouringCarriages.esp")'
+      - <<: *patch3rdParty
+        subs: 
+          - 'Realm of Lorkhan - Freeform Alternate Start'
+          - '[Alternate Perspective - Realm of Lorkhan Patch](https://www.nexusmods.com/skyrimspecialedition/mods/104545/)
+        condition: 'file("Realm of Lorkhan - Custom Alternate Start - Choose your own adventure.esp") and not file("SHE_LorkhanPerspective.esp")'
```

```suggestion
      - <<: *patch3rdParty
        subs:
          - 'Realm of Lorkhan - Freeform Alternate Start'
          - '[Alternate Perspective - Realm of Lorkhan Patch](https://www.nexusmods.com/skyrimspecialedition/mods/104545/)'
        condition: 'file("Realm of Lorkhan - Custom Alternate Start - Choose your own adventure.esp") and not file("SHE_LorkhanPerspective.esp")'
      - <<: *patchUnavailable
        subs: [ 'Touring Carriages' ]
        condition: 'active("TouringCarriages.esp")'
```

There was a missing closing single quote in the second sub causing the CI to fail. Also, this will sort the message aliases alphanumerically & clean up a trailing space.

---

👍  Nice catch, sorry about that, first time doing this stuff